src/e2e/java/teammates/e2e/pageobjects/StudentProfilePage.java

```diff
@@ -159,12 +159,24 @@ public void showPictureEditor() {
         waitForUploadEditModalVisible();
     }
 
-    public void verifyPhotoSize(String height, String width) {
-        assertEquals(height, browser.driver.findElement(By.className("profile-pic")).getCssValue("height"));
-        assertEquals(width, browser.driver.findElement(By.className("profile-pic")).getCssValue("width"));
+    public void closePictureEditor() {
         click(uploadEditModal.findElement(By.className("close")));
     }
 
+    public void verifyPhotoSize(int height, int width) {
+        String browserHeight = browser.driver.findElement(By.className("profile-pic")).getCssValue("height");
+        String browserWidth = browser.driver.findElement(By.className("profile-pic")).getCssValue("width");
+        /*
+         * Remove the last two chars from the string, so we can isolate the Measurement from
+         * "[MEASUREMENT]px", since we want to parse it into a float
```

Comment:
Having this comment doesn't change the fact that `2` is still a magic number. Why not do something more self-explanatory like:
```java
browserHeight.replaceFirst("px$", "")
```

Reply:
Because I actually didn't know that replaceFirst() exists, thanks